### DISCUSSION TITLE
How do you change the top padding of tabs?

### DISCUSSION THREAD

**Discussion by ghost** [black7375/Firefox-UI-Fix#747] [Q&A] [answered]
Hello, I wish to change the top padding of tabs using the lepton theme.

How my tabs currently look: 
![image](https://github.com/black7375/Firefox-UI-Fix/assets/25581533/f995aca1-6045-42e0-b9fd-6e4f0ca34683)

From what I can tell, the padding for the top of the tab appears to be set to 1 pixel, i want to know if its possible to set it to something higher like 2 or 3.

Thank you for reading this, have a nice day.

**Comment by Aleksandra0056** [comment]
What OS are you using?

**Reply by ghost** [reply]
I am running bleeding edge Gentoo Linux, firefox 115.2

**Comment by black7375** [comment, marked as answer]
https://github.com/black7375/Firefox-UI-Fix/wiki/Options#drag-space

![image](https://github.com/black7375/Firefox-UI-Fix/assets/25581533/bf9f30a4-d5cd-41ce-8b0e-91ae7c4bd53e)

You can set `userChrome.padding.drag_space` to `true`!!

**Reply by ghost** [reply]
I deleted my other comment because I had to set the `.maximized` variable of it to true for it to work since I use a tiling window manager. My mistake.
